pcsx2/CDVD/CsoFileReader.h

```diff
@@ -57,6 +57,7 @@ class CsoFileReader final : public ThreadedFileReader
 	bool DecompressFrame(Bytef* dst, u32 frame, u32 readBufferSize);
 	bool DecompressFrame(u32 frame, u32 readBufferSize);
 
+	bool m_uselz4 = false; // flag to enable LZ4 decompression (ZSO files)
```

Comment:
Move this to under the u32 (you'll get 3 extra bytes of padding otherwise)

Reply:
Shouldn't the compiler already do this? With private members there's no guarantee of the ordering if I remember right.

Reply:
To answert my own question: no it doesn't reorder private members, at least not with the default flags.